src/DocumentFormat.OpenXml.Framework/Packaging/OpenSettings.cs

```diff
@@ -12,6 +12,7 @@ public class OpenSettings
     {
         private bool? _autoSave;
         private MarkupCompatibilityProcessSettings? _mcSettings;
+        private bool? _ignoreExceptionsOnCalcChainPartMissing;
```

Comment:
This line is unnecessary

Reply:
again, following the same pattern as _autoSave. Might be better to leave them for readability but will wait for more comments.

Reply:
Unless there are reasons not to, I would remove both private fields and update both to the new syntax

Reply:
VS gives other suggestions for conciseness (i.e. using compound statements) as well for existing code, like the if statement in MarkupCompatibilityProcessSettings.

Reply:
I don't know all of what VS suggests, but I would be on board with modernizing syntax in general